service/qos/service_level_controller.cc

```diff
@@ -152,22 +160,27 @@ future<> service_level_controller::update_service_levels_from_distributed_data()
                     //removed, but only if it is not static since static configurations dont
                     //come from the distributed keyspace but from code.
                     if (!current_it->second.is_static) {
+                        sl_logger.info("service level \"{}\" was deleted.", current_it->first.c_str());
```

Comment:
nitpick: what's all these explicit `c_str()` calls? Can't current_it->first be printed correctly without calling `c_str()`?